ckan/controllers/group.py

```diff
@@ -314,8 +314,14 @@ def pager_url(q=None, page=None):
 
             facets = OrderedDict()
 
-            default_facet_titles = {'organization': _('Organizations'),
-                                    'groups': _('Groups'),
+            org_type = h.default_group_type(u'organization') + u's'
+            org_label = h.humanize_entity_type(org_type).capitalize()
+
+            group_type = h.default_group_type(u'group') + u's'
+            group_label = h.humanize_entity_type(group_type).capitalize()
```

Comment:
Not `.titile()` as you used for package_type?  Even better, maybe make it part of `humanize_entity_type`?